Makefile

```diff
@@ -51,23 +51,24 @@ ner-small-from-parser:
 	python scripts/train_ner.py \
 	--model_output_dir ${SMALL_NER} \
 	--data_path ${MED_MENTIONS} \
-	--model_path ${SMALL_PARSER} \
+	--model_path ${SMALL_PARSER}/best/ \
 	--iterations 7 \
 	--label_granularity 7
 
 ner-large-from-parser:
 	python scripts/train_ner.py \
 	--model_output_dir ${LARGE_NER} \
 	--data_path ${MED_MENTIONS} \
-	--model_path ${LARGE_PARSER} \
+	--model_path ${LARGE_PARSER}/best/ \
 	--iterations 7 \
 	--label_granularity 7
 
 package:
 	# Create model packages for 1) The library and 2) The Spacy model.
 	bash scripts/create_model_package.sh ${BUILD_DIR}
 
-all-small: init-small parser-small ner-small-from-parser package
+all-small: init-small parser-small ner-small-from-parser
```

Comment:
do these commands need to have the package command at the end? or is that still tbd

Reply:
still tbd, I think i'm going to bin the makefile approach, it's too unwieldy so this will change again before we release the models